components/mdx/HighlightMarkdown/HighlightMarkdown.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import Markdown from 'markdown-to-jsx'
+import MDXcomponents from '../../../helpers/mdxComponents'
+
+type Props = {
+  children: string
+  wrapper?: string
+  [key: string]: any
+}
+
+const HighlightMarkdown: React.FC<Props> = ({ children, ...props }) => {
+  return (
+    <Markdown
+      {...props}
+      options={{
+        overrides: {
+          pre: props => {
+            const propsOfChildren = props.children.props
+            const children = propsOfChildren.children
+            const language = propsOfChildren?.className?.split('-')[1]
+
+            return (
+              <MDXcomponents.code className={language || ''}>
```

Comment:
If no language is provided, it'll not highlight the code and it'll look something like:


```
const text = "Hi, this code is not highlighted"
```